pkg/readiness/testdata/99-modifyset.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: mutations.gatekeeper.sh/v1alpha1
+kind: ModifySet
+metadata:
+  name: demo
+spec:
+  applyTo:
+    - groups: [""]
+      versions: ["v1"]
+      kinds: ["Pod"]
+  match:
+    scope: Namespaced
+    kinds:
+      - apiGroups: ["*"]
+        kinds: ["Pod"]
+  location: "spec.some.set"
+  parameters:
+    using:
```

Comment:
this should be updated to values.fromList right

Reply:
ack, will fix on next push.